clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -1488,12 +1488,247 @@ TEST_F(TokenAnnotatorTest, RequiresDoesNotChangeParsingOfTheRest) {
 
 TEST_F(TokenAnnotatorTest, UnderstandsAsm) {
   auto Tokens = annotate("__asm{\n"
-                         "a:\n"
-                         "};");
-  ASSERT_EQ(Tokens.size(), 7u) << Tokens;
+                         "\"a\":\n"
+                         ": x\n"
+                         ":};");
```

Comment:
Is the unquoted `a:` in assembly code invalid? If not, please add your test instead of editing the existing one.

Reply:
AFAIK it's not valid, unless `a` is a CPP string.